.github/workflows/build.yaml

```diff
@@ -222,7 +222,7 @@ jobs:
         working-directory: ${{env.RouterBuildDir}}
         run: |
           ulimit -c unlimited
-          ctest --timeout 1200 -C ${BuildType} -V -T Test --no-compress-output -I ${{matrix.shard}},,${{matrix.shards}} -j2 ${{env.RouterCTestExtraArgs}}
+          ctest --timeout 1200 -C ${BuildType} -V -T Test --no-compress-output -I ${{matrix.shard}},,${{matrix.shards}} -j100 ${{env.RouterCTestExtraArgs}}
```

Comment:
I think we should set this to j12, not j100

Reply:
Ubuntu GHA jobs seem significantly more stable than the Fedora ones. Partially it is because Fedora runs TSan, which check has high overhead.

Starting all ctest targets at once (which this in effect does) results only in small number of test failures.